accessibility/analyze.py

```diff
@@ -0,0 +1,112 @@
+parser.add_argument("--points_buffer_distance", type=int, default=4000,
+                    help="buffer point geometries to account for walking")
+
+# Pick the middle UTM zone
+
+
+def set_utm_from_gdf(gdf):
+    # DEFINE A PROJECTION HERE
+    # Option 1: https://pyproj4.github.io/pyproj/stable/examples.html#find-utm-crs-by-latitude-and-longitude
+    # Option 2: https://timothyrenner.github.io/talks/20171128-geo-pydata/#/transverse-mercator
+    # print(box(*denmark.total_bounds))
+    # https://pyproj4.github.io/pyproj/stable/api/aoi.html#pyproj.aoi.AreaOfInterest
+    bounds = gdf.total_bounds
+    aou = AreaOfInterest(west_lon_degree=bounds[0], south_lat_degree=bounds[1],
+                         east_lon_degree=bounds[2], north_lat_degree=bounds[3])
+    zones = query_utm_crs_info(area_of_interest=aou)
+    middle = zones[int(len(zones)/2)]
+    print("Central zone projection code: "+middle.code)
+    return gdf.to_crs(epsg=middle.code)
+
+
+def analyze(args):
+    print("Reading input file "+args.input)
+    isochrones_gdf = gpd.read_file(
+        args.input)[:args.limit] if args.limit else gpd.read_file(args.input)
+    isochrones_gdf = set_utm_from_gdf(isochrones_gdf)
+    print(f'Buffering geometry by {args.buffer_distance} meters')
+    isochrones_gdf.geometry = isochrones_gdf.geometry.buffer(
+        args.buffer_distance)
+
+    if args.points:
+        points = args.points
+        print("Reading input points "+points)
+        is_json = "json" in Path(
+            points).suffix or "geojson" in Path(points).suffix
+        is_csv = ".csv" == Path(points).suffix
+        if not is_json and not is_csv:
+            print("Invalid points file format, json or csv required")
+            return
+
+        if is_csv:
+            df = pd.read_csv(points)[
+                :args.limit] if args.limit else pd.read_csv(points)
+            if not ('lat' in df.columns and 'lon' in df.columns):
+                print("Points file missing lat and lon columns")
+                return
+            points_gdf = gpd.GeoDataFrame(df.drop(['lon', 'lat'], axis=1),
+                                          crs={'init': 'epsg:4326'},
+                                          geometry=[Point(xy) for xy in zip(df.lon, df.lat)])
+        elif is_json:
+            points_gdf = gpd.read_file(
+                points)[:args.limit] if args.limit else gpd.read_file(points)
+
+        print("Total features: "+str(len(points_gdf)))
+        # project to same crs as isochrones for projection
+        points_buffer_gdf = points_gdf.to_crs(isochrones_gdf.crs)
+
+        print(
+            f'Buffering point gemetry by {args.points_buffer_distance} meters')
+        points_buffer_gdf.geometry = points_buffer_gdf.buffer(
+            args.points_buffer_distance)
+        isochrones_gdf = pd.concat([isochrones_gdf, points_buffer_gdf])
+
+    isochrones_dissolved = isochrones_gdf.dissolve().to_crs(epsg=4326)
+
+    if not args.no_outputs:
```

Comment:
I'd propose we just make this 
```
if args.output:
  save_things()
```

Also clever use of file name matching, folks can just save to whichever format they want. With one layer the difference isn't as notable.